mycore-acl/src/main/resources/components/acl/config/messages_de.properties

```diff
@@ -4,12 +4,12 @@ mcr.accessKey.button.no                        = Nein
 mcr.accessKey.button.remove                    = Entfernen
 mcr.accessKey.button.update                    = Update
 mcr.accessKey.button.yes                       = Ja
-mcr.accessKey.error.collision                  = Der Wert des Zugriffschl\u00FCssels kollidiert mit einem anderen Schl\u00FCssel.
+mcr.accessKey.error.collision                  = Das Secret des Zugriffschl\u00FCssels kollidiert mit einem anderen Schl\u00FCssel.
```

Comment:
Maybe we should translate "Secret" into "Geheimnis" or "Chiffre"?
I looked into https://de.wikipedia.org/wiki/Verschl%C3%BCsselung#Glossar
and https://de.wikipedia.org/wiki/Geheimtext

It's difficult, because we are mixing access keys with cryptography vocabulary: 
"Der verschlüsselte Schlüssel"